Sources/App/Pages/Home.swift

```diff
@@ -1,49 +1,37 @@
-//
-//  home.swift
-//  App
-//
-//  Created by Sven A. Schmidt on 17/10/2018.
-//
+import Plot
 
-import Foundation
 
-
-func homePage() -> HTML.Node {
+func homePage() -> HTML {
     let title = "Swift Package Index"
-    return .html([
+
+    return HTML(
         myhead(title: title),
-        .body([
-            container([
-                row([
-                    .h2([.text(title)]),
-                    ]),
+        .body(
+            container(
+                row(.h2(.text(title))),
                 reconcileButton(),
-                ingestButton(),
-            ]),
-        ])
-    ])
+                ingestButton()
+            )
+        )
+    )
 }
 
 
-func reconcileButton() -> HTML.Node {
-    return .form([.action => "/packages/run/reconcile", .method => "get"], [
-        row([
-            .input([.class => "btn btn-primary", .type => "submit", .value => "Reconcile"])
-        ]),
-        row([
-            .label(["Reconcile the master package list with the Package Index."]),
-        ]),
-    ])
+func reconcileButton() -> Node<HTML.BodyContext> {
+    .form(
+        .action("/packages/run/reconcile"),
+        .method(.get),
+        .input(.class("btn btn-primary"), .type(.submit), .value("Reconcile")),
```

Comment:
This has mostly translated well, except  I couldn't manage for the life of me to nest the entities like before: `form` -> `div class=row` -> `input`. As you can see in the snapshot image that messes up the spacing.

Obviously the looks of this initial page are completely irrelevant and I'm not even sure if my original html is the right way to do it. Just wanted to mention it in case that might be a problem further down the line.

Also, it shows how the snapshots can be useful to see accidental changes.

It might make sense to skip them though until the the design is somewhat stable. I'm adding some notes below how to manage them.

Reply:
I've just spent some time looking at this and it's certainly not happy with it. I can get a `div` and other elements inside a form, but as soon as I put an input inside that div, I get:

```
Referencing static method 'input' on 'Node' requires the types 'HTML.BodyContext' and 'HTML.FormContext' be equivalent
```

My guess is that there's an allowed list of tags, which isn't actually correct as you can pretty much put anything inside a `form`.

This is very much a common problem with HTML DSLs in my experience! Programmers want to put logic into them that doesn't exist in HTML itself. I may be wrong, and I haven't investigated properly yet, but that's my current theory.

Reply:
Yeah, exactly. That’s the error I got and I couldn’t find a way to make the types lines up. The contexts just don’t support it - it’s the same as with swift-html, just a bit further in... meh.

Not sure if this is something we need to worry about. I'm guessing the layout can be fixed to look the same even without that nesting but the difficulties may be harbingers of things to come?

_I_ would be stuck here and revert to Leaf or the previous "non-typed" DSL but that's mainly because nesting `div`s is my hammer for HTML layout 🤣